tests/TestCase/ExceptionsTest.php

```diff
@@ -93,11 +86,6 @@ public function testPersistenceFailedException()
         $this->assertSame(100, $exception->getCode());
         $this->assertSame($previous, $exception->getPrevious());
         $this->assertSame($entity, $exception->getEntity());
-
-        $exception = new PersistenceFailedException(new Entity(), 'message', null, $previous);
```

Comment:
This is the same test as above but checking the default error code. There is no need to do this for a random exception.